packages/react/src/components/UIShell/components/styles/_side-nav.scss

```
@@ -48,12 +48,22 @@ div:has(.#{$prefix}--header)
     font-weight: 600;
   }
 
+  .#{$prefix}--side-nav__icon > svg,
   .#{$prefix}--side-nav__submenu-chevron > svg {
     fill: $icon-primary;
   }
 }
 
-//----------------------------------------------------------------------------
+.#{$prefix}--side-nav__link,
+.#{$prefix}--side-nav__submenu {
+  &:hover {
+    .#{$prefix}--side-nav__icon > svg,
+    .#{$prefix}--side-nav__submenu-chevron > svg {
+      fill: $icon-primary;
+    }
+  }
+}
```

```suggestion
.#{$prefix}--side-nav__submenu:hover {
  .#{$prefix}--side-nav__icon > svg,
  .#{$prefix}--side-nav__submenu-chevron > svg {
    fill: $icon-primary;
  }
}
```

super nitpicky, but we try to avoid nesting more than once in general in scss files in Carbon.